src/devices/Pn532/P7.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Pn532
+{
+    /// <summary>
+    /// The GPIO state of the GPIO located on the P7 port
+    /// Those ports can be used as external GPIO ports when SPI is not used
+    /// Refer to the detailed documentation page 79 and 80
+    /// </summary>
+    [Flags]
+    public enum Port7
```

Comment:
seems like all ports have following naming conventions:

PXY
X=port number (X=1 => Port1, X=2 => Port2 ...)
Y=pin number
`PXY=1 << y;`

would it make sense to collapse them into one?

Reply:
i.e.: `byte ReadPortA()`

Reply:
Yes, there is a pattern but not all sub ports are accessible. Only the ones in the enum are. So to reduce errors and make sure the function will be used correctly, I preferred to use an enum. 
Also keep in mind that reading the values for those 2 ports are done at once in the same function. Implementing something that would read every single port would then call the function multiple times vs only 1 time with the current implementation. And what does cost time is to read or write. So better to group the operations at once.
Also, there will be very very few persons using this feature. It's only if you build your own hardware, where you specifically wants to use the extra GPIO thru this device. I guess when used on an embedded hardware where you can run .NET Core, you won't really need those extra GPIO. Only scenario I saw is when a package device with a relay or equivalent is attached to the PN532 and in this case, you basically ave to control it thru the PN532.
So to keep thing fast and simple, I would stick to the current solution. If we see a need in the future to have individual read or write functions, it will be quite fast to add.